docs/user_guides/fs/storage_connector/creation/bigquery.md

```diff
@@ -59,13 +62,14 @@ it a unique **name** and an optional
    upload from your local using `Upload new file` or choose an existing file within project using `From Project`.
 4. Read Options: There are two ways to read via BigQuery, using the **BigQuery Table** or **BigQuery Query** option:
    
-      1. The table option reads directly from the BigQuery table reference. In the UI set the below fields, 
+      1. **BigQuery Table** - can be used for reading data from BigQuery. You can't create an External FG using this option. The table option reads directly from the BigQuery table reference. In the UI set the below fields, 
          1. *BigQuery Project*: The BigQuery project
          2. *BigQuery Dataset*: The dataset of the table
          3. *BigQuery Table*: The table to read
-      2. The second option is to read by executing a SQL query at runtime, by selecting **BigQuery Query** and setting,
-         1. *Materiliazation Dataset*: Temporary dataset used by BigQuery for writing
-         
+      2. **BigQuery Query** - can be used for both reading data and **creating External FG**. This option executes a SQL query at runtime.
```

Comment:
```suggestion
      2. **BigQuery Query** - This option executes a SQL query at runtime. It can be used for both reading data and **creating external Feature Groups**. 
```